hack/test-go.sh

```diff
@@ -256,18 +256,13 @@ runTests() {
   # must make sure the output from parallel runs is not mixed. To achieve this,
   # we spawn a subshell for each parallel process, redirecting the output to
   # separate files.
-  printf "%s\n" "${@}" | xargs -I{} -n1 -P${KUBE_COVERPROCS} \
-    bash -c "set -o pipefail; _pkg=\"{}\"; _pkg_out=\${_pkg//\//_}; \
-        go test ${goflags[@]:+${goflags[@]}} \
-          ${KUBE_RACE} \
-          ${KUBE_TIMEOUT} \
-          -cover -covermode=\"${KUBE_COVERMODE}\" \
-          -coverprofile=\"${cover_report_dir}/\${_pkg}/${cover_profile}\" \
-          \"${KUBE_GO_PACKAGE}/\${_pkg}\" \
-          ${testargs[@]:+${testargs[@]}} \
-        | tee ${junit_filename_prefix:+\"${junit_filename_prefix}-\$_pkg_out.stdout\"} \
-        | grep \"${go_test_grep_pattern}\"" \
-      && test_result=$? || test_result=$?
+
```

Comment:
what do you mean "only one is passed"? on my linux machine I see multiple instances of hack/run-go-test.sh being called, though each time it's only passed one of the packages (as expected).

Slightly edited for brevity:
```console
$ ps ax | grep run-go-test
 2387 pts/18   S+     0:00 xargs -0 -I{} -n1 -P4 bash -c .../kubernetes/kubernetes/hack/run-go-test.sh {}
 2388 pts/18   S+     0:00 /bin/bash .../kubernetes/kubernetes/hack/run-go-test.sh cluster/addons/dns/kube2sky
 2390 pts/18   S+     0:00 /bin/bash .../kubernetes/kubernetes/hack/run-go-test.sh cmd/hyperkube
 2391 pts/18   S+     0:00 /bin/bash .../kubernetes/kubernetes/hack/run-go-test.sh cmd/kube-apiserver/app
 2478 pts/18   S+     0:00 /bin/bash .../kubernetes/kubernetes/hack/run-go-test.sh cmd/kube-apiserver/app/options
```